docs/use-the-network/coverage-mapping/coverage-mapping.mdx

```diff
@@ -5,5 +5,18 @@ sidebar_label: Coverage Mapping
 slug: /use-the-network/coverage-mapping
 ---
 
-## Coverage Mapping
+# Coverage Mapping
 
```

Comment:
We should have an info box at the top here that tells people we have a dedicate coverage mapping channel, and tha they should join it.